homeassistant/components/obihai/sensor.py

```diff
@@ -35,46 +32,61 @@
 )
 
 
-def setup_platform(
+# DEPRECATED
+async def async_setup_platform(
```

Comment:
I don't think you need to make this async anymore

Reply:
Does it hurt to have it run async?  Just makes sure the Obihai setup isn't blocking, and it goes away after the deprecation sunset...